lib/SyTest/Identity/Server.pm

```diff
@@ -747,9 +748,9 @@ sub get_access_token
    return $ID_ACCESS_TOKEN;
 }
 
-=head2
+=head2 name
 
-   $self->name():
+   $server->name():
```

Comment:
s/:/;/

(fyi: worth noting that parens after a fn call are optional in perl, so you can write this as `$server->name;`)

Reply:
![mind blown](https://user-images.githubusercontent.com/1342360/64533970-7e16c300-d30c-11e9-8884-530d53f3e3be.gif)